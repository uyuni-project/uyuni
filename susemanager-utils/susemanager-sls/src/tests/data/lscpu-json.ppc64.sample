Architecture:            ppc64le
  Byte Order:            Little Endian
CPU(s):                  8
  On-line CPU(s) list:   0-7
Model name:              POWER9 (architected), altivec supported
  Model:                 2.2 (pvr 004e 0202)
  Thread(s) per core:    8
  Core(s) per socket:    1
  Socket(s):             1
  Physical sockets:      2
  Physical chips:        1
  Physical cores/chip:   4
Virtualization features: 
  Hypervisor vendor:     pHyp
  Virtualization type:   para
Caches (sum of all):     
  L1d:                   64 KiB (2 instances)
  L1i:                   64 KiB (2 instances)
NUMA:                    
  NUMA node(s):          1
  NUMA node1 CPU(s):     0-7
Vulnerabilities:         
  Itlb multihit:         Not affected
  L1tf:                  Mitigation; RFI Flush, L1D private per thread
  Mds:                   Not affected
  Meltdown:              Mitigation; RFI Flush, L1D private per thread
  Spec store bypass:     Mitigation; Kernel entry/exit barrier (eieio)
  Spectre v1:            Mitigation; __user pointer sanitization, ori31 speculation barrier enabled
  Spectre v2:            Mitigation; Indirect branch cache disabled, Software link stack flush
  Srbds:                 Not affected
  Tsx async abort:       Not affected
