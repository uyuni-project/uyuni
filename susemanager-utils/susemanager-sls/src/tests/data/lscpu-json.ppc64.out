{
    "cpu_model": "POWER9 (architected), altivec supported",
    "cpu_numanodes": "1",
    "cpu_cores": "1",
    "cpu_sockets": "1",
    "cpu_sum": "8",
    "cpu_threads": "8"
}
